### PR TITLE
Provide --credentials when creating registry

### PR DESCRIPTION
**Pull request by chmouel**

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -96,8 +96,9 @@ NOTE: Replace "linux/amd64" with the appropriate value for your platform/archite
 
 10.  Launch another terminal, change into the same directory you started OpenShift, and deploy the private docker registry within OpenShift with the following commands:
 
-        $ sudo chmod +r openshift.local.config/master/admin.kubeconfig
-        $ ./oadm registry -n default --config=openshift.local.config/master/admin.kubeconfig
+        $ sudo chmod +r openshift.local.config/master/admin.kubeconfig openshift.local.config/master/openshift-registry.kubeconfig
+        $ ./oadm registry -n default --config=openshift.local.config/master/admin.kubeconfig \
+        	--credentials=openshift.local.config/master/openshift-registry.kubeconfig
```
Comment (liggitt) [inline CONTRIBUTING.adoc]: --credentials is deprecated and not needed any more. The service account can be used for API calls as well

Comment (chmouel) [inline CONTRIBUTING.adoc]: ah thanks and what about the serviceaccount ? It does not seem to be created (or documented)

Comment (liggitt) [inline CONTRIBUTING.adoc]: What version are you using?

Comment (chmouel) [inline CONTRIBUTING.adoc]: fresh checked-out origin from this morning compiled localy

Comment (liggitt) [inline CONTRIBUTING.adoc]: If you omit --credentials, the service account is created

**Comment by chmouel** [comment]
Great thanks and sorry for the noise :)

**Pull request closed without merging**